src/dotnet/project.json

```diff
@@ -64,10 +62,10 @@
     }
   },
   "frameworks": {
-    "netstandardapp1.5": {
+    "netstandard1.5": {
       "imports": [
         "dnxcore50",
-        "portable-net45+win8"
```

Comment:
We need this import for DiaSymReader that we use in TestingAbstractions to read full pbds. Why did you remove it here?